streams/delimiter_stream.ts

```diff
@@ -20,8 +53,12 @@ export class DelimiterStream extends TransformStream<Uint8Array, Uint8Array> {
   #matchIndex = 0;
   #delimLen: number;
   #delimLPS: Uint8Array;
+  #disp?: DelimiterDisposition;
 
-  constructor(delimiter: Uint8Array) {
+  constructor(
+    delimiter: Uint8Array,
+    options?: { disposition?: DelimiterDisposition },
```

Comment:
I'd rather prefer if the type declaration was an interface instead of inlined

Reply:
Fixed!